pipeline/06-upload.R

```diff
@@ -0,0 +1,320 @@
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+# 1. Setup ---------------------------------------------------------------------
+#- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+# NOTE: This script requires CCAO employee access. See wiki for S3 credentials
+# setup and multi-factor authentication help
+
+# Load libraries and scripts
+suppressPackageStartupMessages({
+  library(arrow)
+  library(aws.s3)
+  library(aws.ec2metadata)
+  library(dplyr)
+  library(glue)
+  library(here)
+  library(knitr)
+  library(lubridate)
+  library(paws.analytics)
+  library(paws.application.integration)
+  library(tidyr)
+  library(yaml)
+})
+source(here("R", "helpers.R"))
+
+# Initialize a dictionary of file paths. See misc/file_dict.csv for details
+paths <- model_file_dict()
+
+# Load the parameters file containing the run settings
+params <- read_yaml("params.yaml")
+
+# Load various overridden parameters as defined in the `finalize` step
+metadata <- read_parquet(paths$output$metadata$local)
+cv_enable <- metadata$cv_enable
+shap_enable <- metadata$shap_enable
+run_id <- metadata$run_id
+run_type <- metadata$run_type
```

Comment:
Since these attributes are generated in `05-finalize.R`, we have to load them from the output of that stage now.